zio-http/src/main/scala/zhttp/service/server/ServerChannelInitializer.scala

```diff
@@ -47,7 +47,7 @@ final case class ServerChannelInitializer[R](
 
     // ObjectAggregator
     // Always add ObjectAggregator
-    pipeline.addLast(OBJECT_AGGREGATOR, new HttpObjectAggregator(cfg.maxRequestSize))
+    // pipeline.addLast(OBJECT_AGGREGATOR, new HttpObjectAggregator(cfg.maxRequestSize))
```

Comment:
delete commented code, check what other handling is done by Object Aggregator and handle it in getBodyAsString